app-misc/recoll/recoll-1.36.2.ebuild

```
@@ -0,0 +1,128 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
```

Please drop removed targets next time (I'll fix on merge)

```suggestion
PYTHON_COMPAT=( python3_{10..11} )
```